### PR TITLE
Next larger element tree using cpp

### PR DESCRIPTION
**Pull request by anshika272**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given a generic tree and an integer n. Find and return the node with the next larger element in the tree i.e. find a node with a value just greater than n.
Note: Return NULL if no node is present with a value greater than n.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #6706

### CODE REVIEW

**Comment by anshika272** [comment]
@ankitaggarwal23 @HarshCasper @anushkrishnav please review my PR.

File: C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/*Given a generic tree and an integer n. Find and return the node with the next larger element in the tree
+i.e. find a node with a value just greater than n.
+Note: Return NULL if no node is present with a value greater than n.
+*/
+
+#include <iostream>
+
+#include <queue>
+
+#include <vector>
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp]: Replave header with` #include<bits/stdc++.h>`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by anshika272** [comment]
@ankitaggarwal23 pls review the changes

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by ankitaggarwal23** [comment]
@all-contributors bot please add @anshika272 for code

**Pull request merged**